### PR TITLE
1352-oneContributionPerRoleNew

### PR DESCRIPTION
**Pull request by TobiasNx**
See #1352 and #1282

### CODE REVIEW

File: src/main/resources/alma/alma.fix [hunk 1]
```diff
@@ -19,13 +19,12 @@ paste("id", "~https://lobid.org/resources/", "001", "~#!", join_char: "")
...
 
 include ("./fix/titleRelatedFields.fix")
-include ("./fix/contribution.fix")
 include ("./fix/describedBy.fix")
 include ("./fix/fields.fix")
 include ("./fix/subjects.fix")
 include ("./fix/item.fix")
 include ("./fix/mediumAndType.fix")
-
+include ("./fix/contribution.fix")
```
Comment (dr0i) [inline src/main/resources/alma/alma.fix:27]: I like alphabetical ordering, but as we don't force this and it is already unordered, I am gonna ignoring this.

**Review by dr0i — APPROVED** [review]
looks good!

File: src/main/resources/alma/alma.fix (hunk 1 above)
Comment (TobiasNx) [reply src/main/resources/alma/alma.fix:27]: This is necessary due to dependencies for https://github.com/hbz/lobid-resources/issues/1282

Comment (dr0i) [reply src/main/resources/alma/alma.fix:27]: Ah - you mean the ordering _does_ matter ? Ok, I see.

**Pull request merged**